src/shardeum/shardeumFlags.ts

```diff
@@ -122,6 +122,7 @@ interface ShardeumFlags {
   failedStakeReceipt: boolean // For stake/unstake TXs that fail the checks in apply(), create an EVM receipt marked as failed
   debugDefaultBalance: string
   disableSmartContractEndpoints: boolean
+  stakeLockTime: number
```

Comment:
Just realized we need to migrate this from a shardeum flag to being a network parameter
(NetworkParameters)

Reply:
this is required so that we can change the setting in a live network.  flags are great for debug, but not good for persistent configs